mixins/localize-mixin.js

```diff
@@ -124,6 +124,9 @@ export const LocalizeMixin = dedupeMixin(superclass => class extends superclass
 	_generatePossibleLanguages() {
 		const langs = new Set();
 
+		langs.add('en');
```

Comment:
Moved this block before the `docLang` and `docFallbackLang` blocks as those seem to also add `en-us` - which causes the same issue as before.

Reply:
Hmmm. Ok, I don't think we can do that, as it defeats the purpose of checking the document settings. Let me think about this a little bit more.

Reply:
Thanks :) Yeah it seems to be causing issues with the unit tests as well - I'm poking around

Reply:
Related to this - do we need the other `en` lang options here: 
https://github.com/BrightspaceUI/intl/blob/30c5729f28eb856a15c55bb36eecf314a2efa6bb/lib/common.js#L3

This is where those document language settings are pulled from.

Reply:
Okay so I think I have something that works - on the assumption that we don't use `en` variants in web components

Thoughts? @bearfriend @dlockhart @svanherk 

Thanks :)

Reply:
I'm not as familiar with our localize mixins, so someone feel free to jump in - but couldn't we definitely theoretically need an `en-us.js` file?  If a langterm included "favourite" in it and we need an `en-us` file just for those terms?
Also, why is this something that would only happen in dev?  I figure we're seeing this because the LMS defaults to `en-us`, and wouldn't that be a valid setting for all our US customers?

Reply:
Yeah tackling the issue from that side would work as well - adding `en-us` and `en-gb` language files to all web components.

As for why this only happens in Dev - I'm not sure - do the web components have access to a `en-us` in Prod environments? If so that would explain a lot.